src/main/scala/com/github/unchama/seichiassist/util/BreakUtil.scala

```diff
@@ -147,26 +147,32 @@ object BreakUtil {
     }
   }
 
-  def isProtectedNetherQuartzBlock(player: Player, checkTarget: Block): Boolean = {
-    checkTarget.getType match {
-      // 鉱石ブロックの方はプロテクトの対象外
-      case Material.QUARTZ_BLOCK | Material.QUARTZ_STAIRS =>
-        if (
-          !SeichiAssist
-            .instance
-            .breakSkillTargetConfigSystem
-            .api
-            .breakSkillTargetConfig(player, BreakSkillTargetConfigKey.MadeFromNetherQuartz)
-            .unsafeRunSync()
-        ) {
-          ActionBarMessageEffect(s"${RED}スキルでのネザー水晶類ブロックの破壊は無効化されています")
-            .run(player)
-            .unsafeRunSync()
-          true
-        } else {
-          false
-        }
-      case _ => false
+  /**
+   * ブロックがネザー水晶類破壊フラグの保護対象かどうか判定する。
+   * ここでいう「ネザー水晶類」に鉱石は含まれない。
+   * @param player プレイヤー
```

Comment:
単に`プレイヤー`とするとドキュメントが提供している情報がないに等しいので、あくまで一例ですが
```suggestion
   * @param player ネザー水晶類破壊設定を取得するプレイヤー
```

のようになっていると良いと思いました。